code/game/objects/items/weaponry.dm

```diff
@@ -470,6 +470,15 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	desc = "A bust of the famous Greek physician Hippocrates of Kos, often referred to as the father of western medicine."
 	icon_state = "hippocratic"
 
+/obj/item/statuebust/attack_self(mob/living/user)

```

Comment:
why not on examinate

Reply:
you mean in addition to? (for some dumb reason statues are weapons)

Reply:
like #43995 does for spiritual comfort, you might make it a var checked when an item is examinated so any kind of item could be impressive, like examinating an golden cup

Reply:
If you want to do something like that, best would be wrapping it in a component.